app/imaging/preview.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package imaging
+
+import (
+	"bytes"
+	"fmt"
+	"image"
+	"image/jpeg"
+
+	"github.com/disintegration/imaging"
+)
+
+// GenPreview generates the preview for the given image.
+func GenPreview(img image.Image, width int) image.Image {
```

Comment:
Not a fan of shortening function names for exported functions but 0/5

Reply:
It just saved some typing. I am okay either way as well.